bookkeeper-server/src/main/java/org/apache/bookkeeper/client/LedgerHandle.java

```diff
@@ -322,7 +329,17 @@ public void close()
 
         explicitLacFlushPolicy.stopExplicitLacFlush();
 
-        SynchCallbackUtils.waitForResult(counter);
+        SyncCallbackUtils.waitForResult(counter);
+    }
+
+    @Override
+    public CompletableFuture<?> asyncClose() {
```

Comment:
close should be implemented with asyncClose()

Reply:
done